x/distribution/abci.go

```diff
@@ -25,19 +25,15 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
 
 	// determine the total power signing the block
-	var previousTotalPower, sumPreviousPrecommitPower int64
+	var previousTotalPower int64
 	for _, voteInfo := range req.LastCommitInfo.GetVotes() {
 		previousTotalPower += voteInfo.Validator.Power
-		if voteInfo.SignedLastBlock {
-			sumPreviousPrecommitPower += voteInfo.Validator.Power
-		}
 	}
 
 	// TODO this is Tendermint-dependent
 	// ref https://github.com/cosmos/cosmos-sdk/issues/3095
```

Comment:
does this comment still make sense?

Reply:
Yes, it’s identical to the original code. We have a unit test to detect any discrepancies.